### ISSUE TITLE
Deployment path to AWS account: jh-msel-pass-development (229357306566) as pre-staging environment

### ISSUE THREAD

**Issue by dkriethof** [eclipse-pass/main#998]

**Comment by steven-exarcos** [comment]
@rpoet-jh Yes -- I think this as a work item would better be handled on the Jira side, since product development will likely never touch the 'dev' account.   All of what exists currently in 'dev' is terraform output and is used as a testing sandbox.

I do have a [branch](https://github.com/jhu-library-devops/JHU-PASS-Deployment/tree/dev-deployment-sm) in the devops deploy repo to add that account to the cicd pipeline.  But this is mainly in anticipation of doing QA on the terraform stack before it replaces what exists in staging.

Once the migration to terraform is complete, the only use for 'pass-dev' afterwards would be e.g. for one of us to debug a breaking change _before_ it alters the staging environment.

**Comment by rpoet-jh** [comment]
@steven-exarcos Ok, I think I understand.  This ticket is for figuring out how to integrate terraform into the PASS CI/CD pipeline.  I'll create the ticket on the Jira side.

Re: if we keep the terraform `dev` account and make it an official PASS development environment, we can table that for now and discuss more if needed in the future.

I am going to delete ticket 574.
